packages/synth.queues/src/services/kafka.queue.service.ts

```diff
@@ -0,0 +1,135 @@
+import { MultivestError, PluginManager } from '@applicature/synth.plugin-manager';
+import * as config from 'config';
+import * as kafka from 'node-rdkafka';
+import * as logger from 'winston';
+import { Errors } from '../errors';
+import { Message, Queue } from '../types';
+import { QueueService } from './queue.service';
+export class KafkaQueueService extends QueueService {
+    private producerReady: Promise<any>;
+    private consumerReady: Promise<any>;
+
+    constructor(pluginManager: PluginManager) {
+        super(pluginManager);
+    }
+
+    public async createQueue(queueName: string): Promise<Queue> {
```

Comment:
for creating/deleting queues (topics) you should use InternalAdminClient which may create and delete topics